docs/source/en/perf_train_cpu.md

```diff
@@ -18,10 +18,11 @@ rendered properly in your Markdown viewer.
 This guide focuses on training large models efficiently on CPU.
 
 ## Mixed precision with IPEX
+Mixed precision is the use of both single (FP32) and half-precision (BF16/FP16) data types in a model to accelerate training or inference while still preserving much of the single-precision accuracy. Modern CPUs such as 3rd Gen Intel® Xeon® Scalable processors and 4th Gen Intel® Xeon® Scalable processors support the lower-precision data type BFloat16 natively, so you should get performance benefits out of the box by simply turning the mixed precision training with BFloat16 on.
 
-IPEX is optimized for CPUs with AVX-512 or above, and functionally works for CPUs with only AVX2. So, it is expected to bring performance benefit for Intel CPU generations with AVX-512 or above while CPUs with only AVX2 (e.g., AMD CPUs or older Intel CPUs) might result in a better performance under IPEX, but not guaranteed. IPEX provides performance optimizations for CPU training with both Float32 and BFloat16. The usage of BFloat16 is the main focus of the following sections.
+To further maximize training performance, you can use Intel® Extension for PyTorch (IPEX), which is a library built upon PyTorch and adds additional CPU ISA level support such as Intel® Advanced Vector Extensions 512 (Intel® AVX-512) Vector Neural Network Instructions (VNNI) and Intel® Advanced Matrix Extensions (Intel® AMX) for an extra performance boost on Intel CPUs. However, it is not guaranteed that CPUs with only AVX2 (e.g., AMD CPUs or older Intel CPUs) have better performance under IPEX.
```

Comment:
```suggestion
To further maximize training performance, you can use Intel® Extension for PyTorch (IPEX), which is a library built on PyTorch and adds additional CPU instruction level architecture (ISA) level support such as Intel® Advanced Vector Extensions 512 (Intel® AVX-512), Vector Neural Network Instructions (VNNI), and Intel® Advanced Matrix Extensions (Intel® AMX) for an extra performance boost on Intel CPUs. However, CPUs with only AVX2 (e.g., AMD or older Intel CPUs) are not guaranteed to have better performance under IPEX.
```

Reply:
"Intel® Advanced Vector Extensions 512 (Intel® AVX-512) Vector Neural Network Instructions (VNNI)" should be "Intel® Advanced Vector Extensions 512 Vector Neural Network Instructions (Intel® AVX512-VNNI)" VNNI is a vector neural network instruction, which is a subset of AVX512 ISA. Thx @stevhliu 's help.